embuilder.py

```diff
@@ -48,7 +48,7 @@
     'libemmalloc-memvalidate',
     'libemmalloc-verbose',
     'libemmalloc-memvalidate-verbose',
-    'libgl',
+    'libGL',
```

Comment:
This change means that people using embuilder might notice the change? Then a changelog mention might be good.

Reply:
Done